source/docs/articles/sites/code/ldap-and-ldaps.md

```diff
@@ -10,9 +10,9 @@ keywords: ldap, ldaps, ldap and ldaps, using ldap as a provider, ldap provider,
 
 ## LDAP as a Consumer  
 
-Using LDAP as a consumer of services is supported on the platform and will work at all plan levels, assuming correct configuration. The implementation and configuration details are up to the user as not all instances are supported.
+Using LDAP as a consumer of services is supported on the platform and will work at all plan levels, assuming correct configuration. The implementation and configuration details are up to the user as not all instances are supported. For general information about implementing LDAPS, see [https://drupal.org/node/1404368](https://drupal.org/node/1404368) and [https://drupal.org/node/1302032](https://drupal.org/node/1302032).
 
-LDAP provides access and maintenance of a distributed directory storing organized sets of records. LDAP is often used for single sign-on authentication, and many sites on Pantheon use LDAP and LDAPS for authentication. Authentication requests will originate from Pantheon to the LDAP server used (outbound from Pantheon).  
+LDAP provides access and maintenance of a distributed directory storing organized sets of records. LDAP is often used for single sign-on authentication and authentication requests will originate from Pantheon to the LDAP server used (outbound from Pantheon). However, we do not recommend using LDAP for single sign-on authentication as it is not secure.  The recommended approach is SAML (both for sites and the Dashboard) and OneLogin. Learn more about [two-factor authentication](/docs/guides/two-factor-authentication/).

```

Comment:
They would need an IdP provider to use LDAP. But that content already existed and I don't think we want to remove it or give them further instructions on using LDAP for SSO. IMO it doesn't warrant the warning treatment.

If I understood the issue correctly, we just want to make it clear we recommend SAML. Maybe @ari-gold or @bmackinney can clarify this.